src/numerics/include/metaphysicl/sparsenumberbase.h

```diff
@@ -20,112 +20,87 @@
 // Boston, MA  02110-1301  USA
 //
 //-----------------------------------------------------------------------el-
-//
-// $Id: core.h 37197 2013-02-21 05:49:09Z roystgnr $
-//
-//--------------------------------------------------------------------------
-
 
-#ifndef METAPHYSICL_DYNAMICSPARSENUMBERBASE_H
-#define METAPHYSICL_DYNAMICSPARSENUMBERBASE_H
+#pragma once
```

Comment:
Can we switch this back to the usual guard idiom?  ```#pragma once``` does seem pretty universally supported, but it's not an official part of the standard.